ui/src/router/versions/details/RouterVersionDetailsView.js

```diff
@@ -119,7 +119,11 @@ export const RouterVersionDetailsView = ({
 
             <Router primary={false}>
               <Redirect from="/" to="details" noThrow />
-              <RouterVersionConfigView path="details" config={version} />
+              <RouterVersionConfigView
+                path="details"
+                projectId={projectId}
```

Comment:
You likely don't need this `projectId` here, as `@reach/router` will pass it for you